src/nilss.jl

```diff
@@ -289,20 +290,19 @@ function store_y_w_vstar!(y, w, vstar, sol, nus, numindvar, numparams, iseg)
   copyto!(_y, (@view sol[1:numindvar,:]))
 
   # fill w
+  # only calculate w one time, w can be reused for each parameter
   for j=1:nus
-    for i=1:numparams
-      indx1 = (j-1)*numindvar*numparams + i*numindvar+1
-      indx2 = (j-1)*numindvar*numparams + (i+1)*numindvar
+   indx1 = (j-1)*numindvar*1 + 1*numindvar+1
```

Comment:
I think the `1*` can go away and instead of (1+1) I would directly write 2 for better readability.